cmd/syncthing/gui.go

```diff
@@ -233,13 +235,15 @@ func (s *apiService) Serve() {
 		}
 	}
 
-	if listener == nil {
+	if 0 == len(listeners) {
```

Comment:
This is not C, no need for yoda notation.

Reply:
Fine